packages/conf-perl-packages/conf-perl-packages.1/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
```

Comment:
would it be possible to split this into two packages: `conf-perl-string-shellquote` and `conf-perl-ipc-system-simple` ?
This way the name of the new package correctly reflects what it installs (which is rather vague with the current name/setup)

Reply:
100% with you, will do.  I didn't do so, only b/c .... well, so many packages (I figured maybe it'd be less troublesome to the maintainers with fewer packages).